Dockerfile

```diff
@@ -1,10 +1,21 @@
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.0-BETA.3
```

Comment:
Is it correct to rely on the beta version for the prod stuff?

Reply:
AppInsights evolves at slow peace and we needed to start to get all features we needed.

At current we already relay on `applicationinsights-agent-2.4.0-BETA-SNAPSHOT.jar`, this PR just bumps to another BETA.

Generally we faced no issue with this approach in the past, obviously I am not saying we won't in the future.